### DISCUSSION TITLE
creating secret ia a failure

### DISCUSSION THREAD

**Discussion by surendraalamuru22** [learndevopsonline/learndevopsonline#1858] [Q&A]
![Uploading image.png…]()

**Comment by chaithanya1812** [comment]
can you check once, Image is not loaded well..

**Reply by surendraalamuru22** [reply]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/159264263/91e712a5-dda3-42e1-916e-a2d61838953c)

**Comment by chaithanya1812** [comment]
HI surender,
secret-key created 
## issue you used small letters in your command
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/86b526a5-bcc6-4d62-acd5-a2cf78ecab84)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/16aea3f0-a74b-4dd3-b7f2-7acdc8fe72b5)

**Reply by surendraalamuru22** [reply]
but iam unable to fetch secret
![image](https://github.com/learndevopsonline/learndevopsonline/assets/159264263/747418e3-2e07-4313-8b8b-eb9dc23c7ba5)

**Comment by chaithanya1812** [comment]
Hi surender, This job  needs to be runs on workstation

**Reply by surendraalamuru22** [reply]
its showing workstation is offline 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/159264263/bf41ebc4-9fff-423c-ac88-f79278444896)
iam not sure wheather my syntax is right one or not

**Comment by chaithanya1812** [comment]
### share your jenkins credentials

**Reply by surendraalamuru22** [reply]
admin
admin123

**Comment by surendraalamuru22** [comment]
not clear
